bin/utils/python_scripts/filepath_settings.py

```diff
@@ -43,9 +43,9 @@ class Settings:
         'explanation': 'redacted',
         'address': 'redacted',
         'bundlefilenameprefix': 'EA 50111 2023',
-        'casenamehmctsinternal': 'redacted',
-        'hmctscasenameinternal': 'redacted',
-        'appellantnamefordisplay': 'redacted',
+        'casenamehmctsinternal': 'redacted redacted',
+        'hmctscasenameinternal': 'redacted redacted',
+        'appellantnamefordisplay': 'redacted redacted',
```

Comment:
In the new value 'redacted redacted'?

Reply:
Yeah Kleo asked me to do it like that as they are full names. Shouldn't matter in the import anyway but just for knowing they're full names I suppose